src/box/sql/vdbe.c

```diff
@@ -1463,6 +1463,35 @@ case OP_Map: {
 	break;
 }
 
+/**
+ * Opcode: Getitem P1 P2 P3 * *
+ * Synopsis: r[P2] = value[P3@(P1 - 1)]
+ *
+ * Get an element from the value in register P3[P1 - 1] using values in
+ * registers P3, ... P3 + (P1 - 2).
+ */
+case OP_Getitem: {
+	int count = pOp->p1 - 1;
```

Comment:
Why not store into p1 the right count during vdbe creation? Do you really need to do `- 1` at runtime?

Reply:
True, they fixed it. Previously, `count` was the total number of expressions, now it is the number of expressions in brackets, i.e. now the value to which the operator is applied is not counted.